packages/web-client/src/helpers/resource/functions.ts

```diff
@@ -1,4 +1,11 @@
-import { urlJoin } from 'web-pkg/src/utils'
+import { basename } from 'path'
+import { urlJoin } from '../../utils'
+import { DavPermission, DavProperty } from '../../webdav/constants'
+import { Resource } from './types'
+
+const fileExtensions = {
```

Comment:
Why did you move `fileExtensions` to `functions` instead of `types`? 🤔

Reply:
Probably because I moved the functions going along with it here, it's not exported after all.

Have no strong preference. If you prefer it in `types` I can move it there, but it's not really a "type", is it? (I mean technically yes, but nothing you'd use in an interface or something like that)

Reply:
Hmm true, not exported. Then it's fine